client/src/hooks/useMyDataset.js

```diff
@@ -77,7 +79,8 @@ export const useMyDataset = () => {
     const newDataset = datasetRequest.response
 
     setMyDataset(newDataset)
-    setUserFormat(myDataset.format) // Update user preference to match dataset format
+    setMyDatasetFormat(newDataset.format) // Use to compare format changes on myDataset replace
```

Comment:
Why do we call `setMyDataset` and then `setMyDatasetFormat` when presumably format should already be updated with `setMyDataset`. Based on my read we should just be updating the format local storage value inside of `setMyDataset`